pkgs/tools/admin/exoscale-cli/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, buildGo114Package, substituteAll, fetchgit }:
+
+let
+  version = "1.12.0";
+  rev = "4b82b898b56ea238eb80ac38da40dc8a460a7de7";
+in
+buildGo114Package {
+  name = "exoscale-cli-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/exoscale/cli.git";
+    rev = "${rev}";
+    sha256 = "04ym7mfv565icj3lmd2nrvq9asawwmmzg09681pj9py61ws56bxr";
+    fetchSubmodules = true;
+  };
+
+  goPackagePath = "github.com/exoscale/cli";
+  subPackages = [ "." ];
+  goDeps = ./deps.nix;
+
+  # can't use buildFlags here because these are passed to "go install"
+  # which does not recognize -o
+  postBuild = ''
+    mv go/bin/cli go/bin/exo
+  '';
+
+  meta = {
+    description = "Command-line tool for everything at Exoscale: compute, storage, dns";
+    homepage = https://github.com/exoscale/cli;
+    license = stdenv.lib.licenses.asl20;
+    maintainers = [ "Sebastian Krohn <sebastian.krohn@exoscale.ch>" ];
```

Comment:
Aren't you in `maintainers/maintainer-list.nix`?

Reply:
this meta key is supposed to be the "downstream" maintainer - you.

Reply:
I'm not and that is me. :-)

Reply:
Well then, 1st of all welcome aboard :). 2ndly, you should add yourself as a maintainer to `maintainers/maintainer-list.nix`, use https://caius.github.io/github_id/ to get your GitHub id, and make sure that eventually whatever the reviews are, you have 2 commits in this PR:

```
maintainers: add dramaturg
exoscale: init at 1.12.0
```

Oh and use:

```nix
    maintainers = with stdenv.lib.maintainers; [ dramaturg ];
```

Reply:
The bot is not happy with that. Do I need to create a separate PR for that?